### ISSUE TITLE
Save-on-exit prompt option

### ISSUE THREAD

**Issue by Joe-Hu-Chat** [maoschanz/drawing#635]
## Requested feature

If the tab content is not saved, there will always be a prompt to save it when closing that tab. In some cases, works have been copied to clipboard and pasted to other files, for a word file, so there is no need to save that image. I use this great program a lot and clicking the discard button every time makes feel that why don't we have an option to disable this save-on-exit prompt mechanism. So I added this option and recompiled the source code myself. 
Just wonder it's only me needing this feature? If other people need this feature I can upload my change and do some fixes if needed.

### Possible drawbacks

An extra option will exist on the preference menu.

### Possible alternatives

Just click the discard button every time?

## Use case

I would disable this save-on-exit option when most of my use cases are not saving images in Drawing. In other cases, enable it.

**Comment by Joe-Hu-Chat** [comment]
Yes, I use drawing to edit captures when making notes. What I need in this case is copy them to clipboard, rarely I will need to save the pictures.

**Comment by maoschanz** [comment]
if there was a single dialog, listing all your unsaved files, and you could click a "discard all" option, would it meet your need?

**Comment by Joe-Hu-Chat** [comment]
This could relieve me of clicking discard for every unsaved tab by just one click, but I prefer getting rid of the prompt as I don't need it in my use case now and if I changed my habit I could bring it back in preferences setting. 

To clarify, I was not requesting to add this feature, I had already implemented a switch in preferences settings and it worked, then I wanted to see whether this was needed for other people. I just migrated from Windows to Ubuntu a couple months ago and I needed a tool like MS paint and I found drawing. Thank you a lot for making this software! I'm not experienced in making this kind of software, so if this feature is not appropriate, please feel free to close this issue.
